### DISCUSSION TITLE
Can't run dev server and Jest tests concurrently

### DISCUSSION THREAD

**Discussion by davidgruar** [vercel/next.js#15397] [Help]
# Bug report

## Describe the bug

If Jest is running in watch mode, starting the next.js dev server throws an error and also causes the Jest process to abort.

## To Reproduce

1. Start a new project with `npx create-next-app`
2. Install Jest:
    1. run `yarn add jest --dev`
    2. add `"test": "jest --watch"` to `scripts` in `package.json`
    3. add a basic test, e.g. in `app.test.js`: `test('I can run tests', () => expect(true).toBe(true))`
3. Run `yarn test`
4. In a separate console window, run `yarn dev`
5. The process aborts with `Error: ENOTEMPTY: directory not empty, rmdir 'C:\path\to\my\app\.next\server'`
6. The Jest process also aborts with `Error: EPERM: operation not permitted, lstat 'C:\path\to\my\app\.next\static\development\pages'`

The issue also manifests if you are running the Jest extension in VS Code, which automatically runs tests in the background.

## Expected behavior

I should be able to run tests and the app at the same time.

## System information

- OS: Windows 10
- Version of Next.js: 9.4.4
- Version of Node.js: 12.6.1

**Comment by goser** [comment]
If you don't have a problem with putting all your tests into one test folder, moving the tests and the jest config to a subfolder could solve this.
